Fix no side spacing applied in quiz in learning mode for mobile view

Imran92 opened the pull request:
Resolves issue of no side spacing applied in quiz in learning mode for mobile view.

## Proposed Changes
* The lateral spacing for quiz was removed on mobile due to some recent changes. We've fixed that here.
* Lesson spacing was also not applied except for the default template. We've fixed that both on this PR and the linked Pro PR

## Testing Instructions
<!--
Add as many details as possible to help others reproduce the issue and test the changes.
"Before / After" screenshots can also be very helpful when the change is visual.
-->

1. Checkout this branch of Pro https://github.com/Automattic/sensei-pro/pull/2462
2. Check every LM template for both lesson and quiz in mobile and desktop
3. Make sure the spacing on both side is there in all cases

Before:
![Screenshot 2023-10-31 at 3 56 20 PM](https://github.com/Automattic/sensei/assets/6820724/6bd64c75-b662-453e-b615-585de84e8be3)

![Screenshot 2023-10-31 at 4 42 29 PM](https://github.com/Automattic/sensei/assets/6820724/c5a93d25-bed1-43cf-9b50-de73967ad330)

## Pre-Merge Checklist
<!-- Complete applicable items on this checklist **before** merging. Items that are not applicable can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed. -->
- [x] PR title and description contain sufficient detail and accurately describe the changes
- [x] Acceptance criteria is met
- [ ] Decisions are publicly documented
- [ ] Adheres to coding standards ([PHP](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/), [JavaScript](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/), [CSS](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/css/), [HTML](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/html/))
- [ ] All strings are translatable (without concatenation, handles plurals)
- [x] Follows our naming conventions (P6rkRX-4oA-p2)
- [ ] Hooks (p6rkRX-1uS-p2) and functions are documented
- [ ] New UIs are responsive and use a [mobile-first approach](https://zellwk.com/blog/how-to-write-mobile-first-css/)
- [x] New UIs match the designs
- [ ] Different user privileges (admin, teacher, subscriber) are tested as appropriate
- [x] Code is tested on the minimum supported PHP and WordPress versions
- [ ] User interface changes have been tested on the latest versions of Chrome, Firefox and Safari
- [ ] "Needs Documentation" label is added if this change requires updates to documentation
- [ ] Known issues are created as new GitHub issues

Imran92 committed:
Fix spacing not applied on Quiz template

Imran92 committed:
Add changelog

Imran92 committed:
Make inner content styles apply only on default template

donnapep commented on changelog/fix-no-side-spacing-mobile-lm-quiz:
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fixed
+
+Lateral spacing not applied in the mobile view of quiz in learning mode
```
Is this something that currently exists in the 4.18 version? If it's something we introduced in 4.19, then we probably don't need the change log in this PR or the Pro one.

Imran92 replied:
> Is this something that currently exists in the 4.18 version?

It was introduced in 4.19, doesn't exist in 4.18. I've removed the changelog 👍 Thanks! (Also dismissed the review when I removed the changelog, sorry 😄 )

donnapep reviewed (dismissed):
Left a question about perhaps not needing the change log entry.

Also an FYI that I don't believe we're currently handling padding correctly. We should be leveraging [root padding aware alignments](https://developer.wordpress.org/themes/global-settings-and-styles/settings/use-root-padding-aware-alignments/), but I'm not sure that we are. This is another thing that would be nice to look at when we have some time to dig deeper into LM.

Regardless, this fixes the issue so I'm approving.

Imran92 committed:
Revert "Add changelog"

This reverts commit 7468dc15301dddf93d83f8a35bc1422d086472d5.

Imran92 commented:
> Left a question about perhaps not needing the change log entry.

Thanks! Removed the changelog entry as the issue was introduced after the last release.

> Also an FYI that I don't believe we're currently handling padding correctly. We should be leveraging [root padding aware alignments](https://developer.wordpress.org/themes/global-settings-and-styles/settings/use-root-padding-aware-alignments/), but I'm not sure that we are. This is another thing that would be nice to look at when we have some time to dig deeper into LM.

Agreed. We can look into it to see how we can apply it in our LM context.

> Regardless, this fixes the issue so I'm approving.

Thanks! The review got dismissed after I removed the changelog. Can you kindly approve again?
